### PR TITLE
Implementation of Filecoin integration using Estuary interfaces

### PR DESCRIPTION
**Pull request by aaitor**
## Description

Add a description of your changes here.

## Is this PR related with an open issue?

Related to Issue #

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
 
## Checklist:

- [ ] Follows the code style of this project.
- [ ] Tests Cover Changes
- [ ] Documentation

### CODE REVIEW

File: metadata_driver_filecoin/data_plugin.py [hunk 1]
```diff
@@ -1,78 +1,139 @@
...
 from metadata_driver_interface.exceptions import DriverError
-from pygate_grpc.client import PowerGateClient
+from requests import HTTPError
+from requests_toolbelt.multipart import encoder
+
+
+def _process_download(response, local_file=None):
+    try:
+        if local_file is None:
+            new_file, local_file = tempfile.mkstemp()
```
Comment (ssallam) [inline metadata_driver_filecoin/data_plugin.py:16]: `new_file` is a file descriptor, it's probably better to close it.

File: metadata_driver_filecoin/data_plugin.py [hunk 2]
```diff
@@ -130,21 +188,11 @@ class FilecoinUrl:
     cid_hash = ''
     user_token = None
     deal_id = None
-    powergate_host = 'localhost'
-    powergate_port = '5002'
-    is_secure = False
+    gateway_url = 'https://shuttle-4.estuary.tech'
```
Comment (r-marques) [inline metadata_driver_filecoin/data_plugin.py:191]: Should this be retrieved from an env?

**Review by r-marques — APPROVED** [review]
lgtm

File: metadata_driver_filecoin/data_plugin.py (hunk 2 above)
Comment (aaitor) [reply metadata_driver_filecoin/data_plugin.py:221]: it's an option of the constructor so can be passed as parameter, it's just a default value

**Pull request merged**